hrapplications/models.py

```diff
@@ -11,14 +11,22 @@
 
 @python_2_unicode_compatible
 class ApplicationQuestion(models.Model):
-    title = models.CharField(max_length=254)
+    title = models.CharField(max_length=254, verbose_name='Question')
     help_text = models.CharField(max_length=254, blank=True, null=True)
 
     def __str__(self):
         return "Question: " + self.title
 
 
 @python_2_unicode_compatible
+class ApplicationChoice(models.Model):
+    question = models.ForeignKey(ApplicationQuestion,on_delete=models.CASCADE,related_name="choice")
```

Comment:
It would make me much happier to have this related name be pluralized, eg `choices` - otherwise when you use it, you get multiple items by accessing a singular-named attribute which is kinda weird. You know, like if the user groups field was just `group` - `User.group.all()` is funny-sounding.